INSTALL

```diff
@@ -1,288 +1,368 @@
+eventually be run) with '--host=TYPE'.
+
+Sharing Defaults
+================
 
-Information on the location of packages and their versions follows immediately.
+   If you want to set default values for 'configure' scripts to share,
+you can create a site shell script called 'config.site' that gives
+default values for variables like 'CC', 'cache_file', and 'prefix'.
+'configure' looks for 'PREFIX/share/config.site' if it exists, then
+'PREFIX/etc/config.site' if it exists.  Or, you can set the
+'CONFIG_SITE' environment variable to the location of the site script.
+A warning: not all 'configure' scripts look for a site script.
 
-****************** ***
-****   PACKAGES   ****
-****************** ***
+Defining Variables
+==================
 
-For rational polyhedra:
+   Variables not defined in a site shell script can be set in the
+environment passed to 'configure'.  However, some packages may run
+configure again during the build, and the customized values of these
+variables may be lost.  In order to avoid this problem, you should set
+them in the 'configure' command line, using 'VAR=value'.  For example:
 
-CoCoALib, http://cocoa.dima.unige.it/cocoalib/, version 0.99601
-MPFR,  http://www.mpfr.org/mpfr, version 4.0.1
-Flint, http://www.flintlib.org/flint, version 2.5.2
-nauty, http://users.cecs.anu.edu.au/~bdm/nauty, version 27rc2
+     ./configure CC=/usr/local2/bin/gcc
 
-For algebraic polyhedra additionally:
+causes the specified 'gcc' to be used as the C compiler (unless it is
+overridden in the site shell script).
 
-MPFR and Flint as above
-arb, https://github.com/fredrik-johansson/arb/archive/, version 2.16.0
-e-antic, http://www.labri.fr/perso/vdelecro/e-antic/, version 0.1.2
+Unfortunately, this technique does not work for 'CONFIG_SHELL' due to an
+Autoconf limitation.  Until the limitation is lifted, you can use this
+workaround:
 
+     CONFIG_SHELL=/bin/bash ./configure CONFIG_SHELL=/bin/bash
 
-*********************
-****   SCRIPTS   ****
-*********************
+'configure' Invocation
+======================
 
-The Normaliz install scripts handle the installation without any user interaction.
-They only generate directories and files within the Normaliz directory, unless
-the user forces them to do otherwise.
+   'configure' recognizes the following options to control how it
+operates.
 
-The use of the install scripts is described in the Normaliz manual,
-doc/Normaliz.pdf, Section 10.2.
+'--help'
+'-h'
+     Print a summary of all of the options to 'configure', and exit.
 
-They installation can in addition be controlled by several environment variables:
+'--help=short'
+'--help=recursive'
+     Print a summary of the options unique to this package's
+     'configure', and exit.  The 'short' variant lists options used only
+     in the top level, while the 'recursive' variant lists options also
+     present in any nested packages.
 
-NMZ_COMPILER: overrides the standard choice of compiler (g++ on Linux, clang++ on Mac)
```

Comment:
I believe that all of these variables should be handled by the standard switches that `./configure` provides already.